tests/integration-tests/tests/efa/test_efa.py

```diff
@@ -61,9 +61,11 @@ def test_efa(
     remote_command_executor = RemoteCommandExecutor(cluster)
     scheduler_commands = scheduler_commands_factory(remote_command_executor)
 
-    _test_efa_installation(scheduler_commands, remote_command_executor, efa_installed=True, partition="efa-enabled")
-    _test_mpi(remote_command_executor, slots_per_instance, scheduler, scheduler_commands, partition="efa-enabled")
-    logging.info("Running on Instances: {0}".format(get_compute_nodes_instance_ids(cluster.cfn_name, region)))
+    # Run EFA tests if not running OSU benchmark, in future decouple EFA from OSU tests
+    if instance not in osu_benchmarks_instances or os == "rocky8":
```

Comment:
So just for `Rocky8` We are running OSU benchmarks and efa tests too. 
Please add a TODO to remove this condition when we are going to decouple EFA and OSU.

Reply:
Added the todo!